src/views/MyProfile/index.js

```diff
@@ -99,25 +99,21 @@ const MyProfile = props => {
         {!loading && (
           <div className="personal-profile">
             <Grid container justify="center" spacing={2}>
-              {/* START OF IDENTIFICATION CARD */}
               <Grid
                 item
                 xs={12}
                 sm={12}
                 md={profile.PersonType === 'stu' ? 8 : 12}
                 lg={profile.PersonType === 'stu' ? 6 : 10}
               >
-                <Identification profile={profile} width={props.width} network={network} />
+                <Identification profile={profile} network={network} myProf={true} />
               </Grid>
```

Comment:
The width prop was removed because I realized that nothing was actually being passed through in that variable. If you haven't caught it in the files above, this problem was solved by adding a function in the Identification component that gets the width of the screen and checks to see what Material-UI breakpoint the current screen size is at.